.github/workflows/flutter_test.yml

```diff
@@ -54,7 +54,7 @@ jobs:
         run: flutter pub get
 
       - name: Gradle cache
-        uses: gradle/gradle-build-action@982da8e78c05368c70dac0351bb82647a9e9a5d2 # pin@v2.11.1
+        uses: gradle/gradle-build-action@3b1b3b9a2104c2b47fbae53f3938079c00c9bb87
```

Comment:
This is a more generic comment, it would be nice to keep the version comment, not sure when it was removed.

Reply:
yeah, I'll check and attach them